src/components/TextField/TextField.tsx

```diff
@@ -29,7 +34,22 @@ export const TextField: React.FC<Props> = ({
 
   // To show errors only if the field was touched (onBlur)
   const [touched, setTouched] = useState(false);
-  const hasError = touched && required && !value;
+
+  let isError = false;
+  let errorMessage = '';
+
+  // eslint-disable-next-line max-len
+  const pattern = /^((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w-_]*)?\??(?:[-+=&;%@,.\w_]*)#?(?:[,.!/\\\w]*))?)$/;
+
+  if (name === URLs.ImgUrl || name === URLs.ImdbUrl) {
+    isError = touched && required && !pattern.test(value);
+    errorMessage = `Please enter valid URL for ${label}`;
+  }
+
+  if (!value) {
```

Comment:
```suggestion
  if (!value.trim()) {
```
it will fix problem with only empty spaces value
![chrome-capture-2023-6-31](https://github.com/mate-academy/react_movies-list-add-form/assets/33029864/ac128bce-79bb-4ba3-9a4b-aebccf61301d)